components/Templates/includes/functions-view_template.php

```diff
@@ -91,6 +91,12 @@ function frontier_decode_template( $code, $atts ) {
  */
 function frontier_if_block( $atts, $code ) {
 
+	$atts = wp_parse_args( $atts, array(
```

Comment:
* :no_entry_sign: Opening parenthesis of a multi-line function call must be the last content on the line
* :no_entry_sign: Only one argument is allowed per line in a multi-line function call

<sub>hash: cdaf72b4733b1d88d4d2f4489f93544f</sub>